src/main/java/frc/robot/subsystems/drive/DriveSubsystem.java

```diff
@@ -271,10 +271,10 @@ public void drive(double xSpeed, double ySpeed, double rotRate, boolean fieldRel
    * Sets the wheels into an X formation to prevent movement.
    */
   public void setX() {
-    m_frontLeft.setDesiredState(new SwerveModuleState(0, Rotation2d.fromDegrees(45)));
-    m_frontRight.setDesiredState(new SwerveModuleState(0, Rotation2d.fromDegrees(-45)));
-    m_rearLeft.setDesiredState(new SwerveModuleState(0, Rotation2d.fromDegrees(-45)));
-    m_rearRight.setDesiredState(new SwerveModuleState(0, Rotation2d.fromDegrees(45)));
+    m_frontLeft.setDesiredStateAndGetDesiredState(new SwerveModuleState(0, Rotation2d.fromDegrees(45)));
```

Comment:
is the reason we replaced it here to avoid having duplicate methods? It feels odd that we're being given a value back that isn't updating the module state